### DISCUSSION TITLE
Exporting Modified Tab Settings

### DISCUSSION THREAD

**Discussion by MarkTheTestStrategist** [furybee/chrome-tab-modifier#403] [Q&A]
I have setup a lot of tabs with groups and custom icons for a profile on my work laptop, but I want to export the file that holds all this information so I can use it on another computer for the same profile. 

Where can I find the file I need to export please?

**Comment by sylouuu** [comment]
Go to Options > Settings > Export

**Reply by MarkTheTestStrategist** [reply]
Thank you.
